src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colors {
+    white,
+    black,
+    purple,
+    red,
+    yellow,
+    green,
+    blue,
+    brown,
+    orange,
+    pink
```

Comment:
[Follow the naming convention. Enum values are constants.](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)